src/compiler/types.ts

```diff
@@ -3846,6 +3846,7 @@ namespace ts {
     }
 
     export const enum InferenceFlags {
+        None            =      0,  // No special inference behaviors

```

Comment:
I kind of like 0. It’s pretty succinct.

Reply:
But we have `None` for so many other enums - plus, it also clarifies what `0` actually does at the callsite to someone reading the code.